fineract-core/src/main/java/org/apache/fineract/infrastructure/core/service/database/JdbcJavaType.java

```diff
@@ -136,11 +136,19 @@ public static JdbcJavaType getByTypeName(@NotNull DatabaseType dialect, String n
         for (JdbcJavaType type : values()) {
             DialectType dialectType = type.getDialectType(dialect);
             if (dialectType.getNameResolved().equals(name)) {
+                // NOTE: make MySQL systems happy aka TINYINT vs BOOLEAN issue!
```

Comment:
Can you please explain the use-case why do you need this and why you do not change the BOOLEAN and TINYINT behaviour for mysql?

Reply:
I use the code that you created. That's why I asked you for help a couple of weeks ago, because you know your implementation (and the goals you want to achieve with it) best. If you have an alternative that fixes the problem and fits better in your database mapping mechanics then please send me a snippet. Happy to replace.
But for now this is the least intrusive way that I could come up with in 5min. My goal is to create a working release.

Reply:
Have to do the release. Please propose a pull request if you see any need for further improvement. As is works.